content/webapp/pages/stories.tsx

```diff
@@ -86,12 +87,24 @@ const StoryPromoContainer = styled.div.attrs({
   }
 `;
 
+// TODO we need to support old colors (teal/red/green/purple) so fix it better as part of
```

Comment:
Adding this to https://github.com/wellcomecollection/wellcomecollection.org/issues/8540